### PR TITLE
Contact branch

### PR DESCRIPTION
**Pull request by uchexm**
- I created a contact form and linked it to Formspree.
- I Changed some of the tags to use semantic tags
- I Updated the class names to adhere to Html/CSS best practices.
- Hein added form validations,
- And fixed linter errors.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -368,110 +376,187 @@ li:hover {
...
+  padding-bottom: 20px;
+  background: #6070ff;
+  background-image: url("images/contactbg.png");
+  background-repeat: no-repeat;
+  background-position: 180px 20px;
+  margin-top: -98px;
+  visibility: visible;
+  background-size: 200px;
+  border-top-left-radius: 50px;
+}
```
Comment (mckent05) [inline style.css:468]: - According to the project requirements, please use the `flexbox` display property to align items in this section. This allows for more flexibility and responsiveness 🙏

File: index.html [hunk 2]
```diff
@@ -6,168 +6,153 @@
...
-    </ul>
-  </section>      
+    <section class="Contact-me">
+      <h3 id="Me">Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form action="https://formspree.io/f/mqkjgawz" method="post" name="Form" class="Click-1">
+        <input type="text" class="Details" name="name" placeholder="Uchechi" maxlength="30" required>
+        <input type="email" class="Details" name="name" placeholder="nwakauc1@gmail.com" required>
+        <textarea name="Text" id="Details" cols="50" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+        <button id="Get-in-touch" class="button-2" type="submit" target="_blank" action="submit">Get in touch</button>
```
Comment (mckent05) [inline index.html:154]: - You have not applied the right hover styling on your button, please try to fix this. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. You can consider fixing this for other buttons as they may apply 👍 👍 

hover2

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @uchexm,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Contact form is working as expected 👍 
- All linter checks are passing ✔️ ✔️ 
- Descriptive PR title and summary 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**OPTIONAL** - I suggest you make your portfolio responsive across all mobile screens, this would ensure your site displays correctly and fits well on all mobile screens. Please consider fixing this 👍 👍 
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by uchexm** [comment]
I have implemented the requested changes.

File: style.css [hunk 3]
```diff
@@ -227,251 +242,352 @@ li:hover {
...
+  background-color: white;
 }
 
-.skills {
-  display: inline-block;
-  flex-direction: column;
-  flex-wrap: wrap;
-  justify-content: center;
-  align-items: center;
+button:hover {
```
Comment (topeogunleye) [inline style.css:563]: - [ ] As suggested by the previous reviewer, please add the correct background color on hover, , active (when it's pressed).

Screenshot from 2022-04-27 19-12-50

the button on hover has the `background: #6070FF;` and `color: #FFFFFF`, 

**for hover state**

```
button:hover {
    background-color: #6070FF;
    color: #FFFFFF;
}
```

File: style.css [hunk 4]
```diff
@@ -227,251 +242,352 @@ li:hover {
...
-  font-family: "Poppins", serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  color: #000;
-  margin-left: px;
-  display: inline-block;
-  justify-content: space-around;
+button:active {
```
Comment (topeogunleye) [inline style.css:567]: - [ ] As suggested by the previous reviewer, please add the correct background color on active (when it's pressed).

Screenshot from 2022-04-27 19-12-50

The button on the pressed state has the `background: #2230D2;` and `color: #FFFFFF`, 

**for pressed state**
```
button:active {
   background-color: #2230D2;
   color: #FFFFFF;
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @uchexm, and Team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (uchexm) [reply style.css:567]: Aye aye captain

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @uchexm 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**